src/main/java/com/epam/tc/hw8/constants/Constants.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.tc.hw8.constants;
+
+public class Constants {
+
+    public static final String POSITIVE = "positive";
+    public static final String NEGATIVE = "negative";
+
+    public static final String TASK_TRACKER = "Task tracker";
+
+    public static final String GET = "get";
+    public static final String POST = "post";
+    public static final String DELETE = "delete";
+    public static final String PUT = "put";
```

Comment:
Specifying the method's constants is not required, as you can use Method.GET/POST/PUT... enum